src/mds/Server.cc

```diff
@@ -5794,30 +5794,35 @@ int Server::parse_quota_vxattr(string name, string value, quota_info_t *quota)
       }
     } else if (name == "quota.max_bytes") {
       /*
-       * The "quota.max_bytes" must be aligned to 4MB if greater than or
-       * equal to 4MB, otherwise must be aligned to 4KB.
+       * The "quota.max_bytes" must be aligned to 4MiB if greater than or
+       * equal to 4MiB, otherwise must be aligned to 4KiB.
        */
       string cast_err;
       int64_t q = strict_iec_cast<int64_t>(value, &cast_err);
-      if(!cast_err.empty() ||
-         (!IS_ALIGNED(q, CEPH_4M_BLOCK_SIZE) &&
-          (q < CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4K_BLOCK_SIZE)))) {
-        dout(10) << __func__ << ":  failed to parse quota.max_bytes: "
+      if(!cast_err.empty()) {
+        dout(10) << __func__ << ": failed to parse quota.max_bytes: "
                  << cast_err << dendl;
         return -CEPHFS_EINVAL;
       }
+      if (q >= CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4M_BLOCK_SIZE) ||
+          q < CEPH_4M_BLOCK_SIZE && !IS_ALIGNED(q, CEPH_4K_BLOCK_SIZE)) {
+        dout(10) << __func__ << ": bad quota.max_bytes value " << q
+                 << ", which must be aligned to 4KiB if less than 4MiB or must be aligned to 4MiB"
```

Comment:
- `impermissible value << q for quota.max_bytes` or
 `forbidden value << q for quota.max_bytes`
- s/which/it

This is quite trivial, so it's all up to you.

Reply:
I was just following the existing debug log:

`dout(10) << __func__ << ": bad vxattr value, unable to parse int for " << name << dendl;`

let's keep the same with this here.

Reply:
alright